ssl_verify_client.t

```diff
@@ -128,6 +128,8 @@ foreach my $name ('1.example.com', '2.example.com', '3.example.com') {
 		. "-out $d/$name.crt -keyout $d/$name.key "
 		. ">>$d/openssl.out 2>&1") == 0
 		or die "Can't create certificate for $name: $!\n";
+	system("cat $d/$name.crt $d/$name.crt > $d/$name.bundle") == 0
+		or die "Can't create certificate bundle $name.bundle: $!\n";
```

Comment:
This is not portable, we have read_file / write_file routines for that kind of things.

Reply:
Yes, that's right.